src/Firebase.Auth/FirebaseAuthProvider.cs

```diff
@@ -375,6 +399,10 @@ private static AuthErrorReason GetFailureReason(string responseData)
                         case "EMAIL_EXISTS":
                             failureReason = AuthErrorReason.EmailExists;
                             break;
+                            
+                        //possible errors from Account Delete
+                        case "USER_NOT_FOUND":
+                            failureReason = AuthErrorReason.UserNotFound;
```

Comment:
Yeah... I edited directly on GitHub and forgot of some basic things... But I already tested the DeleteUser method and it's working :)